overlord/patch/patch6_2_test.go

```diff
@@ -47,7 +48,7 @@ var statePatch6_2JSON = []byte(`
 			"sequence": [
 			  {
 				"name": "snapd",
-				"snap-id": "snapd-snap-id",
+				"snap-id": "PMrrV4ml8uWuEUDBT8dSGnKUYbevVhc4",
```

Comment:
Same question as above.

Reply:
again multiine string, also these are the tests of a patch, I wanted to touch them as little as possible and in the most mechanical way as possible

Reply:
here other comments in the file explain the magic string, also it can be inferred from "name"